releases/release-1.26/exceptions.yaml

```diff
@@ -0,0 +1,117 @@
+# Exception requests in 1.26
+# Google Group: https://groups.google.com/g/kubernetes-sig-release
+# Release Team Lead: Leonard Vincent Simon Pahlke ([@leonardpahlke](https://github.com/leonardpahlke))
+# Release Team Shadows: Grace Nguyen ([@gracenng](https://github.com/gracenng))
+# / Kat Cosgrove ([@katcosgrove](https://github.com/katcosgrove))
+# / Priyanka Saggu ([@Priyankasaggu11929](https://github.com/Priyankasaggu11929))
+# / Nate Waddington ([@nate-double-u](https://github.com/nate-double-u))
+
+# Enhancements Freeze Exceptions requested in 1.26
+
+enhancementFreeze:
+
+- name: "CCM Webhook Framework"
+  issue: 2699
+  date_requested: 2022-10-07
+  date_reviewed: 2022-10-10
+  thread: "https://groups.google.com/g/kubernetes-sig-release/c/Uw3SpvyfGh0"
+  pull_requests:
+    - "https://github.com/kubernetes/kubernetes/pull/108838"
+  status: "approved"
+
+- name: "Subsecond Probes"
+  issue: 3066
+  date_requested: 2022-10-07
+  date_reviewed: 2022-10-12
+  thread: "https://groups.google.com/g/kubernetes-sig-release/c/P9Uzf3oQaX8"
+  pull_requests:
+    - "https://github.com/kubernetes/enhancements/pull/3067"
+  status: "denied"
+
+- name: "Add AppArmor support"
+  issue: 24
+  date_requested: 2022-10-09
+  date_reviewed: 2022-10-11
+  thread: "https://groups.google.com/g/kubernetes-sig-release/c/ziUKliRqNFM"
+  pull_requests:
+    - "https://github.com/kubernetes/enhancements/pull/3298"
+  status: "approved"
+
+- name: "Dynamic Kubelet Configuration"
+  issue: 281
+  date_requested: 2022-10-10
+  date_reviewed: 2022-10-10
+  thread: "https://groups.google.com/g/kubernetes-sig-release/c/mSgvETUVYn0"
+  pull_requests:
+    - "https://github.com/kubernetes/kubernetes/pull/112643"
+  status: "approved"
+
+# Code Freeze Exceptions requested in 1.26
+
+codeFreeze:
```

Comment:
We are missing one: https://groups.google.com/g/kubernetes-sig-release/c/pwb__uKXWjA/m/fFmG764gBAAJ

Reply:
Updated to add the StatefulSet Slice exception